### PR TITLE
Allows translucent backgrounds to be applied to WebViews + adds props for disabling scrolling

### PR DESCRIPTION
**Pull request by lwansbrough**
Enables overwriting of underlying colors for WebViews. Especially useful if you want to give your WebView a transparent background.

### CODE REVIEW

**Comment by lwansbrough** [comment]
Signed the CLA.

**Comment by nicklockwood** [comment]
Why not just "backgroundColor"? Also, I feel like the "opaque" attribute is an implementation detail that we should hide. For example, we could set it automatically based on the opacity of the color specified.

**Comment by lwansbrough** [comment]
I really have no preferences as to how it's implemented :P I just did it the way it makes sense to me. I used underlayColor so that it isn't confused with the backgroundColor style attribute.

**Comment by nicklockwood** [comment]
I'm a little unclear about why backgroundColor is distinct from underlayColor? Why would you ever want them to be different?

**Comment by lwansbrough** [comment]
@nicklockwood Yeah I regret changing that. I see now also that `backgroundColor` is probably already manipulated through the React `style` prop, so adding `backgroundColor` (and especially (!) `underlayColor` under a different name than the actual property it's editing) is completely redundant.

I agree with your initial comment: hiding the `opaque` property behind the `backgroundColor` setter is best. Just check if the color has opacity and if it does, set `opaque = false`.

**Comment by lwansbrough** [comment]
Changes discussed above have been made. Also added props for disabling scrolling and/or scroll bounce.

File: Libraries/Components/WebView/WebView.ios.js [hunk 1]
```diff
@@ -213,6 +217,8 @@ var RCTWebView = createReactIOSNativeComponentClass({
   validAttributes: merge(ReactIOSViewAttributes.UIView, {
     url: true,
     html: true,
+    allowBounce: true,
```
Comment (vjeux) [inline Libraries/Components/WebView/WebView.ios.js]: can you use the same name as <ScrollView> component? I think it's just `bounces` and `scrollEnabled`

Comment (lwansbrough) [inline Libraries/Components/WebView/WebView.ios.js]: Yup!

**Comment by vjeux** [comment]
Looks good to me after the change, thanks for using backgroundColor, it's much more logical that way

**Pull request closed without merging**